apps/feedahelp-cms/schemas/ingredient.ts

```diff
@@ -0,0 +1,47 @@
+import {LemonIcon} from '@sanity/icons'
+import {defineField, defineType} from 'sanity'
+
+export default defineType({
+  name: 'ingredient',
+  title: 'Ingredient',
+  type: 'document',
+  icon: LemonIcon,
+  fields: [
+    defineField({
+      name: 'ingredientName',
+      title: 'Ingredient Name',
+      type: 'string',
+      validation: (Rule) => Rule.required(),
+    }),
+    defineField({
+      name: 'ingredientNameBangla',
+      title: 'Ingredient Name Bangla',
+      type: 'string',
+      validation: (Rule) => Rule.required(),
+    }),
+    defineField({
+        name: 'ingredientImage',
+        title: 'Ingredient Image',
+        type: 'image',
+        validation: (Rule) => Rule.required(),
+    }),
+    defineField({
+        name:'price',
+        title:'Price',
+        type:'number',
+        validation: (Rule) => Rule.required(),
+    }),
+    defineField({
+        name: 'status',
+        title: 'Status',
+        type: 'boolean',
+        validation: (Rule) => Rule.required(),
+    }),
```

Comment:
We need a new `measuringQty` =>

example:
The units needed for ingredient calculation qty can vary depending on the ingredient and the recipe. However, some common units used include:

- Grams (g)
- Milliliters (mL)
- Ounces (oz)
- Cups (c)
- Tablespoons (Tbsp)
- Teaspoons (tsp)
- Dash
- Pinch
- Piece
- Clove
- Sprig
- Stem